content/en/software/access-online/_index.md

```diff
@@ -25,59 +25,22 @@ Running Interlisp online is good for experimenting and introducing yourself to t
 {{< imgproc Login_Screen Resize "400x450">}} Interlisp Online Login {{< /imgproc >}}
 2. Login to Medley Interlisp Online:
    * You can login as a guest by clicking <img src="Guest_Login_button.png" alt="Guest Login button"> on the login screen. However, guest sessions are not saved.
-   or
    * We suggest you create your own account by clicking <img src="New_User_Register_here_button.png" alt="New User Register here button"> on the login screen. Having an account enables you to save your sessions. To create an account, you just need an email address and password. Click Register here on the login screen to create your own account.
-   or
+
    * If you are already registered (created an account), log in and start a Medley Interlisp session. Sessions are preserved for users that login with their own account. However, user account sessions may be deleted after 30 days of inactivity.
 
-3. Select the Exec you want to run. For this exercise, select `Interlisp`
+3. Select the Exec you want to start out with. For example, select `Interlisp`. 
 
-4. Leave the `Fill browser window` option set.
+4. Leave the `Fill browser window` option set. 
 
 5. Select `Run Medley`.
 Your browser will open a window that represents the Interlisp Desktop and looks much like this:
 
 {{< imgproc Online_Initial_Medley Resize "800x450">}} Medley Interlisp{{< /imgproc >}}
 
-The Interlisp Desktop at startup contains 4 windows of interest:
-
-* Prompt Window: The black window at the top of the screen. It is used to display system or application prompts
-* Exec (INTERLISP) window: The main window where you run functions and develop programs.
-* Medley logo window: A window containing the Interlisp Medley logo as a bit map.
-* Status Bar window
-
-### Writing a sample Interlisp programs
-
-In the Exec window, type the following:
-
-```lisp
-(PLUS 1 1)
-```
-
-When you complete typing the ending `)` the Interlisp interpreter will perform the calculation and return the result.
-
-One thing you probably noticed, the command `PLUS` is capitalized. It’s not that the developers of Interlisp were always shouting at each other. Rather, when Interlisp was developed computer programming was in its infancy and standards for naming commands were still evolving.
-
 <div class="mx-auto">
- <a class="btn btn-lg btn-danger mr-3 mb-4" href="{{< relref "../.." >}}">
-  Guide to help you begin programming in Interlisp<i class="fas fa-arrow-alt-circle-right ml-2"></i>
+ <a class="btn btn-lg btn-danger mr-3 mb-4" href="{{< relref "/software/using-medley" >}}">
+  Guide to Using Medley i class="fas fa-arrow-alt-circle-right ml-2"></i>
```

Comment:
The leading angle bracket was lost:  **<i** class="fas fa-arrow-alt-circle-right ml-2">